webapp/channels/src/components/section_notice.tsx

```diff
@@ -14,8 +14,8 @@ type Button = {
     text: string;
 }
 type Props = {
-    title: string;
-    text: string;
+    title: string | React.ReactElement;
+    text?: string;
```

Comment:
Hi @calebroseland ! I stumbled upon this change, and I have a few comments.

First of all, this component is supposed to follow what is defined here: https://www.figma.com/design/maNEjEdsC2KqMll4KPC5Gc/Components---Section-Notice?node-id=2-315&node-type=canvas&t=y3KummxB6xxeE8mp-0

The changes you have made make it go a bit out of the initial designs. If that is intended, it would be great if the designs are updated accordingly. If not, we should either adapt what we are showing to the design, or use a different component.

Also, I am not a big fan of passing react elements to the title. I see that you are using it to pass a special channel label to the string. I would prefer if we find a better way to handle that. Also, not sure if the initial purpose of this component expected the title to have "special markdown". It would be something to ask UX.

This is already merged, and I am not saying we should revert this change or anything similar, but I would love to discuss this a bit deeper with you so we can get this component back into the rails before it diverts too much.